README.md

```diff
@@ -1 +1,226 @@
-# sonarqube-github-integration
\ No newline at end of file
+# Repository line coverage with sonarqube and github connectors
+
+SonarQube is an open source platform developed by SonarSource for continuous inspection of code quality to perform 
+automatic reviews with static analysis of code to detect bugs, code smells and security vulnerabilities.
+Similarly GitHub brings together the world's largest community of developers to discover, share, 
+and build better software. From open source projects to private team repositories, 
+GitHub is an all-in-one platform for collaborative development.
+
+> This guide will walk you through the process of obtaining the test line coverage of each repository in a 
+github organization.
+
+The following sections are covered in this guide
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Developing the program](#developing-the-program)
+- [Testing](#testing)
+- [Deployment](#deployment)
+ 
+## What you'll build
+
+Let's consider a sample scenario to understand how you can use ballerina connectors. The following scenario specifies 
```

Comment:
It's not that easy to understand the use case. Can you explain it in concise manner and highlight the real world requirement to integrate these two SaaS APIs.